frameworks/CSharp/FastHttpApi/Benchmarks/Program.cs

```diff
@@ -0,0 +1,76 @@
+﻿using BeetleX.FastHttpApi;
+using Microsoft.Extensions.Hosting;
+using System.Threading.Tasks;
+using Microsoft.Extensions.DependencyInjection;
+using System;
+using System.Threading;
+
+
+namespace Benchmarks
+{
+
+    [BeetleX.FastHttpApi.Controller]
+    class Program
+    {
+        private static string mDate;
+        public static void Main(string[] args)
+        {
+            mDate = DateTime.Now.ToUniversalTime().ToString("r");
+            var builder = new HostBuilder()
+                .ConfigureServices((hostContext, services) =>
+                {
+                    services.AddHostedService<BeetleXHttpServer>();
+
+                });
+
+            var result = builder.RunConsoleAsync();
+            result.Wait();
+
+        }
+
+        public object plaintext(IHttpContext context)
+        {
+            context.Response.Header[HeaderTypeFactory.DATE] = mDate;
+            return new TextResult("Hello, World!");
+        }
+        public object json(IHttpContext context)
+        {
+            context.Response.Header[HeaderTypeFactory.DATE] = mDate;
+            return new JsonResult(new JsonMessage { message = "Hello, World!" });
+        }
+
+        public class JsonMessage
+        {
+            public string message { get; set; }
+        }
+
+
+    }
+
+
+    public class BeetleXHttpServer : BackgroundService
```

Comment:
You can just use IHostedService instead of BackgroundService. That way you don't have to spin on the token.

Reply:
Thank you. I'm going to modify it.